docker-compose-examples/grist-local-testing/README.md

```diff
@@ -9,4 +9,12 @@ See https://support.getgrist.com/self-managed for more information.
 
 ## How to run this example
 
-This example can be run with `docker compose up`.
\ No newline at end of file
+Before running, create a directory where Grist will store its documents and settings:
```

Comment:
Nitpicking:
```suggestion
Before running the first time, create a directory where Grist will store its documents and settings:
```

Reply:
As an alternative, may we create the directory ourselves?
```bash
$ touch docker-compose-examples/grist-local-testing/persist/.gitkeep
$ git add -f docker-compose-examples/grist-local-testing/persist/.gitkeep # force the file to be indexed, ignoring .gitignore
```

And also this change to make `PERSIST_DIR` equal to `./persist` in `docker-compose.yml` by default:
```diff
diff --git a/docker-compose-examples/grist-local-testing/docker-compose.yml b/docker-compose-examples/grist-local-testing/docker-compose.yml
index 028d4e0f..5ccf7399 100644
--- a/docker-compose-examples/grist-local-testing/docker-compose.yml
+++ b/docker-compose-examples/grist-local-testing/docker-compose.yml
@@ -3,6 +3,6 @@ services:
     image: gristlabs/grist:latest
     volumes:
       # Where to store persistent data, such as documents.
-      - ${PERSIST_DIR}/grist:/persist
+      - ${PERSIST_DIR:-./persist}/grist:/persist
     ports:
       - 8484:8484
```

What do you think?